src/MorseL/HubWebSocketHandler.cs

```diff
@@ -402,7 +402,8 @@ private Task HandleUnparseableReceivedInvocationDescriptor(Connection connection
                     {
                         var policy = await AuthorizationPolicy.CombineAsync(provider, effectiveAuthorizeData);
 
-                        if (!await authorizationService.AuthorizeAsync(user, context, policy))
+                        var authResult = await authorizationService.AuthorizeAsync(user, context, policy);
```

Comment:
I'm really glad this was the only change necessary to maintain authorization!

Reply:
Yeah, same!